packages/server/index.ts

```diff
@@ -35,10 +38,16 @@ async function startServer() {
     app.use(vite.middlewares);
   }
 
+
+
   if (!IS_DEV) {
     app.use('/assets', express.static(path.resolve(distPath, 'assets')));
   }
 
+  app.use(ROUTER_API_PATH, express.json());
+  // app.use(ROOT_API_FORUM_PATH, forumApiRoutes);
+  app.use(ROUTER_API_PATH, apiRouter);
```

Comment:
я предлагаю собрать все роутеры в один и один подключить здесь, чтобы тут не было 100500 роутов
```suggestion
  app.use('/api', router);
```